server/prometheus.js

```diff
@@ -20,6 +20,31 @@ const monitorCertIsValid = new PrometheusClient.Gauge({
     help: "Is the certificate still valid? (1 = Yes, 0= No)",
     labelNames: commonLabels
 });
+
+const monitorUptime1y = new PrometheusClient.Gauge({
+    name: "monitor_uptime_1y",
+    help: "Monitor Uptime 1y (%)",
+    labelNames: commonLabels,
+});
+
+const monitorUptime30d = new PrometheusClient.Gauge({
+    name: "monitor_uptime_30d",
+    help: "Monitor Uptime 30d (%)",
+    labelNames: commonLabels,
+});
+
+const monitorUptime24h = new PrometheusClient.Gauge({
+    name: "monitor_uptime_24h",
+    help: "Monitor Uptime 24h (%)",
+    labelNames: commonLabels,
+});
+
+const monitorAverageResponseTime = new PrometheusClient.Gauge({
```

Comment:
please make sure that users + developers know that this is 24h ping times by adding the `24h` suffix as above.